crate/operator/create.py

```diff
@@ -389,17 +389,17 @@ def get_statefulset_crate_command(
         for k, v in cluster_settings.items():
             settings[f"-C{k}"] = v
 
-    node_settings = node_spec.get("settings", {})
-    for k, v in node_settings.items():
-        settings[f"-C{k}"] = v
-
     if config.CLOUD_PROVIDER == CloudProvider.AWS:
```

Comment:
Actually, I think this block of conditions needs to be moved before the `if cluster_settings` above (line 388). Otherwise, setting `.spec.cluster.settings['node.attr.zone']` there would still be overridden.

Reply:
Agreed, well spotted thanks.